src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoCategory.java

```diff
@@ -24,6 +24,24 @@ public static void registerRecipes(IModRegistry registry) {
         registry.addRecipes(Lists.newArrayList(
             new MultiblockInfoRecipeWrapper(new PrimitiveBlastFurnaceInfo())
         ), "gregtech:multiblock_info");
+        registry.addRecipes(Lists.newArrayList(
```

Comment:
Did you know that Lists.newArrayLists can accept unlimited number of arguments, so you can put all your definitions into one addRecipesCall?

Reply:
This is what I was saying you were going to yell at me about.